multi/stm32l4-multi/flash.c

```diff
@@ -96,6 +99,15 @@ static void _program_unlock(void)
 }
 
 
+static void _program_optunlock(void)
```

Comment:
Use module name prefix even for static functions

Reply:
Done, although I did not change prefixes for not my functions